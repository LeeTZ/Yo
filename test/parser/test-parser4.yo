a = 42 + 1
