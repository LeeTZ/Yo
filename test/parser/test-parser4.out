INT_MAX = 42 + 1
for i = 0 to v:
dist[i] = INT_MAX
sptset[i] = false
if dist[i]:
continue
elif sptset[i] != false:
break
elif dist[i] < m:
return m
else:while m < 3:
dist[i] = i + 3